### PR TITLE
nvidia-470-glx-driver: Enable per-kernel initrd, add usr-merge changes

### PR DESCRIPTION
**Pull request by Staudey**
**Summary**

This adds the same changes already applied to the glx and beta driver. Minus things like GSP firmware, which is not useful on this old driver branch.

**Test Plan**
- Rebooted
- Launched Steam
- Played the games "Balatro" (Proton) and "Clustertruck" (native)

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by Staudey** [comment]
Oops, forgot one thing, sec

File: packages/n/nvidia-470-glx-driver/pspec_x86_64.xml [hunk 1]
```diff
@@ -126,6 +131,8 @@
             <Path fileType="executable">/usr/bin/nvidia-smi</Path>
             <Path fileType="executable">/usr/bin/nvidia-xconfig</Path>
             <Path fileType="library">/usr/lib64/kernel/cmdline.d/40_nvidia.conf</Path>
+            <Path fileType="library">/usr/lib64/kernel/initrd-com.solus-project.current.6.8.12-293.nvidia</Path>
+            <Path fileType="library">/usr/lib64/kernel/initrd-com.solus-project.lts.6.6.33-247.nvidia</Path>
```
Comment (ReillyBrogan) [inline packages/n/nvidia-470-glx-driver/pspec_x86_64.xml:135]: These files need to go into their respective kernel package

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
See comment

**Comment by Staudey** [comment]
Whoops, good catch. One sec.

**Comment by Staudey** [comment]
Done!

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**